plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExMemory.hs

```diff
@@ -1,40 +1,25 @@
 -- editorconfig-checker-disable-file
-{-# LANGUAGE CPP                   #-}
 {-# LANGUAGE DeriveAnyClass        #-}
-{-# LANGUAGE DerivingVia           #-}
 {-# LANGUAGE FlexibleInstances     #-}
-{-# LANGUAGE LambdaCase            #-}
-{-# LANGUAGE MagicHash             #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE TypeApplications      #-}
-{-# LANGUAGE TypeOperators         #-}
-{-# LANGUAGE UndecidableInstances  #-}
 
 module PlutusCore.Evaluation.Machine.ExMemory
-( CostingInteger
-, ExMemory(..)
-, ExCPU(..)
-, ExMemoryUsage(..)
-) where
-
-import PlutusCore.Data
-import PlutusCore.Pretty
-import PlutusPrelude
+    ( CostingInteger
+    , ExMemory(..)
+    , ExCPU(..)
+    ) where
 
 import Codec.Serialise (Serialise)
-import Control.Monad.RWS.Strict
+import Control.DeepSeq
 import Data.Aeson
-import Data.ByteString qualified as BS
-import Data.Proxy
 import Data.SatInt
-import Data.Text qualified as T
-import GHC.Exts (Int (I#))
-import GHC.Integer
-import GHC.Integer.Logarithms
-import GHC.Prim
+import Data.Semigroup
+import GHC.Generics
 import Language.Haskell.TH.Syntax (Lift)
 import NoThunks.Class
-import Universe
+import Text.Pretty
+import Text.PrettyBy
 
 {-
```

Comment:
The warning below should probably be moved to `ExMemoryUsage.hs`.

Reply:
Oh right, silly me. Thanks for catching!